app/controllers/users.js

```diff
@@ -154,6 +157,36 @@ export function create(req, res) {
   }
 }
 
+export const logIn = (req, res) => {
+  const { email, password } = req.body;
+  if (password && email) {
+    if (validator.isEmail(req.body.email)) {
+      User.findOne({
+        email
+      }).then((existingUser) => {
+        if (!existingUser) {
+          return res.status(404).send({ message: 'User not found' });
```

Comment:
When working with Authentication, especially login, it's a much better approach to send generic error messages, as opposed to specific ones.